mage/query-modules/python/node-similarity.md

```diff
@@ -0,0 +1,210 @@
+---
+id: node_similarity
+title: node_similarity
```

Comment:
I am not able to see changes locally since the name of your file in `sidebarMAGE.js` is `node-similarity`, and not `node_similarity`

Reply:
I think I fixed it.